Tests/Realm.Tests/Realm.Tests.csproj

```diff
@@ -1,35 +1,60 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp3.1;monoandroid90;xamarin.ios10;xamarin.mac20</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);uap10.0.19041</TargetFrameworks>
-    <TargetFrameworks Condition="'$(AddNet5Framework)' == 'true'">$(TargetFrameworks);net5.0</TargetFrameworks>
+    <LocalDev>false</LocalDev>
```

Comment:
Once this is merged, folks using Windows should be able to set it to `true` to exclude all non-net461 targets, thus speeding up VS builds.